src/components/navbar/Navbar.tsx

```diff
@@ -46,36 +90,54 @@ function Navbar() {
 
   return (
     <NavbarStyles>
-      <NavLink path={HOME_PAGE_ROUTE}>
+      <NavLink path={role === "freelancer" ? HOME_PAGE_ROUTE : CLIENT_HOME}>
         <img height="50px" alt="logo" src={logo} />
       </NavLink>
 
       {screenWidth < MOBILE_SCREEN_SIZE ? (
         ""
       ) : (
         <>
-          <NavLink path={SEARCH_WORK_ROUTE}>{t("SearchWork.title")}</NavLink>
-          <NavLink path={PROPOSALS_ROUTE}>{t("Proposals.title")}</NavLink>
-          <NavLink path={MY_JOBS_ROUTE}>{t("MyJobs.title")}</NavLink>
-          <NavLink path={DASHBOARD_ROUTE}>{t("Dashboard.title")}</NavLink>
+          {navLinks.map(({ title, to }) => (
+            <NavLink path={to}>{title}</NavLink>
+          ))}
```

Comment:
Key property missing in the NavLink component, React complains about it in the console